protocol/0079-TGAP-transaction_gas_and_priority.md

```diff
@@ -87,7 +87,7 @@ gas = min((maxGas/minBlockCapacity)-1,batchGas)
 
 ```go
 gasOliq = network.transaction.defaultgas + peg cost factor x pegs
```

Comment:
The LP submission, amendment and cancelation should not produce any orders anymore so why do we still need this?

Reply:
Good point, the gas is the same as for the the regular order submission so doesn't really matter, but I can remove that section.